micov/_io.py

```diff
@@ -166,6 +167,25 @@ def _test_has_header(line):
     return has_header
 
 
+def _test_has_header_taxonomy(line):
+    if isinstance(line, bytes):
+        line = line.decode('utf-8')
+
+    genome_id_columns = ('genome-id', 'genome_id', 'feature-id',
+                         'feature_id')
+    taxonomy_columns = ('taxonomy', 'name', 'full_name')
```

Comment:
Can these be setup as static strings in `_constants` similar to the other column names used?

Reply:
...sorry, actually, jsut include the `COLUMN_GENOME_ID`, the others are as far as I know just one offs.

Same re: the `taxonomy_columns`, include `COLUMN_TAXONOMY` and others are afaik one offs